colibre/scripts/birth_metallicity_redshift.py

```diff
@@ -1,5 +1,5 @@
 """
-Makes a stellar metallicity vs. redshift 2D plot. Uses the swiftsimio library.
+Makes a stellar birth metallicity vs. redshift 2D plot. Uses the swiftsimio library.
```

Comment:
This change (and the script's name change) is just to clarify what the script is doing